Vagrantfile

```diff
@@ -35,6 +35,7 @@ end
 Vagrant.configure("2") do |config|
 
   config.vm.box = "ailispaw/barge"
+  config.disksize.size = "128GB"
```

Comment:
Add a var `$vm_disk = 128GB` on top of the file
and use this to define `config.disksize.size`